Dockerfile

```diff
@@ -1,142 +1,198 @@
-FROM lthub/moodle:3.9.25
+FROM lthub/moodle:4.1.13
 MAINTAINER Tyler Cinkant <tyler.cinkant@ubc.ca>
 
-RUN curl -L https://moodle.org/plugins/download.php/22949/mod_questionnaire_moodle310_2020062302.zip -o /questionnaire.zip \
+##6May2024: production release
+##Version 4.1.9
+##FROM dangtue2020/moodlecore:419stg2-41ec0380b10f 
+
+##Version 4.1.10
+##FROM dangtue2020/moodlecore:4110-2d9b62e726d4 
+
+
+#Updated to newer questionnaire 10Dec2023
+RUN curl -L https://moodle.org/plugins/download.php/29228/mod_questionnaire_moodle42_2022092202.zip -o /questionnaire.zip \
     && cp /questionnaire.zip /var/www/html/mod/ \
     && cd /var/www/html/mod \
     && unzip questionnaire.zip \
-    && rm questionnaire.zip \
+    && rm questionnaire.zip \ 
 
+    #This is the latest version of choicegroup plugin 18Jun2024
+    #REMOVED no confirmation from Kate for production 30July2024
+    #&& curl -L https://moodle.org/plugins/download.php/30464/mod_choicegroup_moodle43_2023110900.zip -o /choicegroup.zip \
+    #&& cp /choicegroup.zip /var/www/html/mod/ \
+    #&& cd /var/www/html/mod \
+    #&& unzip choicegroup.zip \
+    #&& rm choicegroup.zip \
+
+    #This is the latest version of plugin moodle33 as of 20Dec2022 and 11Dec2023
     && curl -L https://moodle.org/plugins/download.php/11565/mod_certificate_moodle33_2016052300.zip -o /certificate.zip \
     && cp /certificate.zip /var/www/html/mod/ \
     && cd /var/www/html/mod \
     && unzip certificate.zip \
     && rm certificate.zip \
 
-    && curl -L https://moodle.org/plugins/download.php/29419/mod_hvp_moodle42_2023061200.zip -o /hvp.zip \
+#Updated to newer hvp 10Dec2023 and 18Apr2024
```

Comment:
You don't need to add this kind of comment. The information can be reflect in the commit and commit message.